G1ANT.Addon.Selenium/G1ANT.Addon.Selenium/SeleniumAddon.cs

```diff
@@ -40,14 +35,15 @@ public override void LoadDlls()
         private void UnpackDrivers()
         {
             var unpackfolder = AbstractSettingsContainer.Instance.UserDocsAddonFolder.FullName;
-            Dictionary<string, byte[]> exeList = new Dictionary<string, byte[]>()
+            var driversDictionary = new Dictionary<string, byte[]>()
             {
                 { "chromedriver.exe", Resources.chromedriver },
                 { "geckodriver.exe", Resources.geckodriver },
                 { "IEDriverServer.exe", Resources.IEDriverServer },
                 { "MicrosoftWebDriver.exe", Resources.MicrosoftWebDriver }
             };
-            foreach (var exe in exeList)
+            foreach (var exe in driversDictionary.Where(e => !File.Exists(Path.Combine(unpackfolder, e.Key))
+            || e.Value.Length != new FileInfo(Path.Combine(unpackfolder, e.Key)).OpenRead().Length))
```

Comment:
`FileInfo` should already contain size of the file, you don't need to open it.
DRY - `Path.Combine(unpackfolder, e.Key)`; move the lambda (or some parts of it) into a method.
`driversDictionary` with `exe` elements => `driversDictionary` => `drivers`, `exe` => `driver`